ops/terraform/services/server/modules/bfd_server_asg/main.tf

```diff
@@ -1,8 +1,8 @@
 locals {
   env = terraform.workspace
 
-  rds_availability_zone = data.external.rds.result["WriterAZ"]
```

Comment:
`local.rds_availability_zone` was _probably_ added in when copying existing patterns from the pipeline terraservice. This is unused.